src/main/kotlin/uk/gov/dwp/dataworks/aws/AwsParsing.kt

```diff
@@ -0,0 +1,42 @@
+package uk.gov.dwp.dataworks.aws
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import org.slf4j.LoggerFactory
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.core.io.ClassPathResource
+import org.springframework.stereotype.Component
+import uk.gov.dwp.dataworks.AwsIamPolicyJsonObject
+import uk.gov.dwp.dataworks.logging.DataworksLogger
+import uk.gov.dwp.dataworks.services.ConfigurationResolver
+
+@Component
+class AwsParsing(){
+    @Autowired
+    private lateinit var  configurationResolver: ConfigurationResolver
+
+    @Autowired
+    private lateinit var awsCommunicator: AwsCommunicator
```

Comment:
Neither of these `@Autowired` vars are needed

Reply:
Removed the @BeforeEach as well, as this wasn't being used either